browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -87,9 +90,10 @@ void assertCanPerformBrowserSwitch(FragmentActivity activity, BrowserSwitchOptio
 
         if (!isValidRequestCode(requestCode)) {
             errorMessage = activity.getString(R.string.error_request_code_invalid);
-        } else if (returnUrlScheme == null) {
-            errorMessage = activity.getString(R.string.error_return_url_required);
-        } else if (!browserSwitchInspector.isDeviceConfiguredForDeepLinking(appContext, returnUrlScheme)) {
+        } else if (returnUrlScheme == null && browserSwitchOptions.getAppLinkUri() == null) {
```

Comment:
This condition was updated to ensure that a `returnUrlScheme` or `appLinkUri` is set.